changelogs/2024-09-12_opt-out-of-fast-branch.mdx

```diff
@@ -0,0 +1,21 @@
+---
+_schema: default
+title: Opt out of Fast Branch
+---
+This release added the option to opt out of Fast Branch.
+
+It also addressed several issues, including those affecting the DAM browser, collections in Eleventy Sites, Multiselect inputs, and tables in the Content Editor.
+
+## Features & Improvements
+
+* Added the option to opt out of Fast Branch.
+
+## Fixes
+
+* Fixed an issue where the DAM browser search bar would remember the last term you searched, filtering future searches.
```

Comment:
I think this arguably sounds like a feature we're removing. The real fix is that the "future searches" would often return different results than what you saw last time.

Maybe something like "Fixed an issue where the DAM browser would unsuccessfully filter to your last search term after opening"?